src/app.js

```diff
@@ -11,8 +12,10 @@ app.get('/', (req, res) => {
   res.send('hi there');
 });
 
-app.use('/posts', postsRoutes);
+app.use('/posts', postsRoutes());
 
 app.listen(port, () => {
   console.log(`running on port http://localhost:${port}`);
 });
+
+app.use('/chatrooms', chatRoomRoutes());
```

Comment:
also from my comment down below :arrow_down: you will need just 1 `app.use` imagine having a huge app with 30 routes would you `app.use()`  30 times ?!!!